force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -62,14 +67,17 @@ public with sharing class ClassesObjectsMethods {
      */
     public static List<Boolean> checkStringContent(String input) {
         // Initialize a list to store the results
-        
+     List<Boolean> stringContentList = new List<Boolean>();   
         // Check if the string starts with 'Sales'
-        
+        Boolean startsWithSales = input.startsWith('Sales');
+        stringContentList.add(startsWithSales);
         // Check if the string ends with 'Force'
-        
+        Boolean endsWithForce = input.endsWith('Force');
+        stringContentList.add(endsWithForce);   
         // Check if the string contains 'Awesome'
-        
-        return null; // Replace null with the variable you used to store the result
+        Boolean containsAwesome = input.contains('Awesome');
+        stringContentList.add(containsAwesome);
+        return stringContentList; // Replace null with the variable you used to store the result
```

Comment:
Like above you could add to the list directly. 

  booleans.add(input.contains('Awesome'));